src/main/pages/che-7/overview/assembly_che-quick-starts.adoc

```diff
@@ -23,52 +23,37 @@ This section contains instructions for getting quickly started with {prod}. Vari
 {prod-short} is available in two modes:
 
 * *Single-user*: Non-authenticated {prod-short}, lighter and suited for personal environments
-
 * *Multi-user*: Authenticated {prod-short} suited for organizations and developer teams
 
 
 .Prerequisites
 
-* Kubernetes (version 1.9 or higher) or OpenShift cluster to deploy {prod-short} on.
+* A Kubernetes (version 1.9 or higher) or OpenShift (version 3.11 or higher) cluster to deploy {prod-short} on.
 
 
 == Deploying {prod-short} on a local single-node cluster
 
 WARNING: Single-node clusters are suited only for testing or single-user development. Do *not* use such clusters to run {prod-short} for organizations or developer teams.
 
 * link:{site-baseurl}che-7/installing-{prod-id-short}-on-codeready-containers/[Installing Che on CodeReady Containers]
```

Comment:
Hi, this is because starting with a gerond is part of the rules we follow in modular docs.

See: https://github.com/eclipse/che-docs/blob/master/CONTRIBUTION.md where there is a mention to Modular Docs.

And more specifically: https://redhat-documentation.github.io/modular-docs/#modular-docs-procedure-examples

I feel sorry that we apply these rules in a semi-automated way. We should be explaining much more to contributors. I hope this answers satify you.

Reply:
Hi. Thanks again!
I totally understand the purpose of having a modular doc and standalone procs.
I would not have renamed the procs but only the texts displayed to have a lighter feeling on this page.